modular_skyrat/modules/ambitions/code/ambition.dm

```diff
@@ -390,7 +390,7 @@
 		return TRUE
 
 /datum/ambitions/proc/auto_approve()
-	message_admins(span_big(span_adminhelp("[ADMIN_TPMONTY(my_mind.current)]'s ambitions were automatically approved")))
+	message_admins(span_adminhelp("[ADMIN_TPMONTY(my_mind.current)]'s ambitions were automatically approved"))
```

Comment:
```suggestion
	message_admins(span_big(span_adminhelp("[ADMIN_TPMONTY(my_mind.current)]'s ambitions were automatically approved")))
```

Reply:
I've confirmed with an Head Admin that they're perfectly fine with the removal of the size increase.